permanent_upload/validation.py

```diff
@@ -0,0 +1,24 @@
+import csv
+import os
+
+
+def validate_supported_types(results, data_file="data/supported_file_types.csv"):
+    validation_dataset = {}
+    data_file_path = os.path.join(
+        os.path.dirname(os.path.realpath(__file__)), data_file
+    )
+    with open(data_file_path, "r") as csvfile:
+        validation_reader = csv.reader(csvfile)
+        for row in validation_reader:
+            validation_dataset[row[0]] = row[1:]
+    for result in results:
+        extension = result[0].split(".")[1]
+        assert validation_dataset[extension]
+        # File type classification (e.g. audio, image)
+        assert validation_dataset[extension][0] == result[1]
+        # Final status (e.g. ok, manual_review)
+        assert validation_dataset[extension][1] == result[2]
```

Comment:
Is `uploaded` also an ok status? Like, should we fail the functional test if the file status after 60 status is `uploaded`?